src/openrouter.py

```diff
@@ -20,13 +20,24 @@ async def llm_call(
         completion = await client.chat.completions.create(
             model=llm_model,
             temperature=0,
+            top_p=0,
+            seed=9999,
+            max_tokens=1000,
             messages=[
                 {
                     "role": "system",
                     "content": prompt_template.format(
-                        OPTIONS=options, INSTRUCTION=instruction, INPUT_TEXT=input_text
+                        OPTIONS=options
                     ),
                 },
+                {
+                    "role": "assistant",
+                    "content": f"<instuctions>{instruction}</instuctions>",
+                },
+                {
+                    "role": "user",
+                    "content": f"<specification>{input_text}</specification>",
+                },
```

Comment:
please explain why to use assistent for instructions instead of system prompt, any reasons for that?

Reply:
Answered above